strawberryfields/backends/fockbackend/ops.py

```diff
@@ -338,9 +344,9 @@ def beamsplitter(t, r, phi, trunc):
     # pylint: disable=bad-whitespace
 
     theta = np.arccos(t)
-    BS_tw, _, _ = BSgate(theta, phi, cutoff=trunc)
+    BS_tw = beamsplitter_tw(theta, phi, cutoff=trunc)
 
-    # TODO: Transpose needed because of different conventions in SF and The Walrus. Remove when The Walrus is updated.
+    # Transpose needed because of different conventions in SF and The Walrus.
```

Comment:
Just curious: so this means that the difference in convention remains with the update in The Walrus, correct? (Slightly connected to similar changes proposed in #378)